crates/ruma-events/src/rtc/notification.rs

```diff
@@ -0,0 +1,204 @@
+//! Type for the MatrixRTC notification event (MSC4075).
+//!
+//! Stable: `m.rtc.notification`
+//! Unstable: `org.matrix.msc4075.rtc.notification`
+
+use js_int::UInt;
+use ruma_common::MilliSecondsSinceUnixEpoch;
+use ruma_events::{relation::Reference, Mentions};
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+/// The content of an `m.rtc.notification` event.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[cfg_attr(not(ruma_unstable_exhaustive_types), non_exhaustive)]
+#[ruma_event(
+    type = "m.rtc.notification",
+    kind = MessageLike
+)]
+pub struct RtcNotificationEventContent {
+    /// Local timestamp observed by the sender device.
+    ///
+    /// Used with `lifetime` to determine validity; receivers SHOULD compare with
+    /// `origin_server_ts` and prefer it if the difference is large.
+    pub sender_ts: MilliSecondsSinceUnixEpoch,
+
+    /// Relative time from `sender_ts` during which the notification is considered valid.
+    pub lifetime: UInt,
```

Comment:
Shouldn't this be a `Duration`? We have helpers for that in `ruma_common::server::duration`.